### PR TITLE
v1.12 release: new hardware, sensor, nuttx and uavcan notes

### PR DESCRIPTION
**Pull request by mrpollo**
* table of contents addition
* adds new supported hardware boards
* adds NuttX release notes
* adds sensor update notes
* adds uavcan notes

### CODE REVIEW

**Comment by mrpollo** [comment]
@hamishwillee let's see how this looks, I'm not sure about the NuttX changes but let's see if it looks ok

**Comment by mrpollo** [comment]
@hamishwillee going to merge and we can fix afterwards, I just want to see how it looks

File: en/releases/1.12.md [hunk 1]
```diff
@@ -22,6 +30,27 @@
 * **Airframe Scripts**
   * The syntax for setting defaults was changed and custom scripts require an update
   * See [PR#16796](https://github.com/PX4/PX4-Autopilot/pull/16796/files#diff-dcf2f5536f47f260e5e0ff3b3fd22eaef6b6c510126463d70affa0eb7bd4d3ddL20) for an example.
+* Safety (switch) defaults to off (motors are disarmed, but servos / flaps can move)
+* Safety switch is latching: Once it is disabled, it will stay disabled
+* Added support for IRC Ghost including telemetry
+
+### Sensors
+* Magnetometer calibration faster and more robust
```
Comment (hamishwillee) [inline en/releases/1.12.md:38]: This is a big deal, and if significantly faster worthy of a blog.

Does the process change at all? Do any of the documented existing parameters change?

File: en/releases/1.12.md [hunk 2]
```diff
@@ -22,6 +30,27 @@
...
+  * new soft iron calibration coefficients
+  * automatically determine the rotation of external sensors
+* Optimized rate control sensor pipeline (minimal inner loop end-to-end latency)
```
Comment (hamishwillee) [inline en/releases/1.12.md:41]: Any impact on docs?

File: en/releases/1.12.md [hunk 3]
```diff
@@ -22,6 +30,27 @@
...
+
+### Hardware
+The release includes new hardware support for the following boards, peripherals, and accessories:
+* Pixhawk FMUv6U (Read more about this spec on the [Pixhawk GitHub Repository](https://github.com/pixhawk/Pixhawk-Standards))
+* Pixhawk FMUv6X (Read more about this spec on the [Pixhawk GitHub Repository](https://github.com/pixhawk/Pixhawk-Standards))
+* CUAV X7 / X7Pro ([Read more about this product on the manufacturers site)(http://www.cuav.net/en/x7en/)]
```
Comment (hamishwillee) [inline en/releases/1.12.md:47]: Any of these boards actually new? I thought Nora and X7 were in v1.11?

Comment (hamishwillee) [reply en/releases/1.12.md:47]: Either way, links should be to our docs rather than manufacturer site.

File: en/releases/1.12.md [hunk 4]
```diff
@@ -22,6 +30,27 @@
...
+* CUAV Nora ([Read more about this product on the manufacturers site)(http://www.cuav.net/en/nora/))
+* CUAV CAN GPS ([Read more about this product on the manufacturers site)(http://www.cuav.net/en/neo-3-2/))
+* SP Racing H7 Extreme ([Read more about this product on the manufacturers site](http://seriouslypro.com/spracingh7extreme))
+* Bitcraze Crazyflie v2.1 ([Read more about this product on the manufacturers site](https://www.bitcraze.io/products/crazyflie-2-1/))
+* ARK CAN Flow ([Read more about this product on the manufacturers site](https://arkelectron.com/product/ark-flow/))
```
Comment (hamishwillee) [inline en/releases/1.12.md:52]: Same, this now has docs

File: en/releases/1.12.md [hunk 5]
```diff
@@ -68,7 +97,7 @@
 
 * **RTL improvements ([PR#16377](https://github.com/PX4/PX4-Autopilot/pull/16377))**
   * Hardens the RTL safety failsafes taking into consideration the many edge cases when trying to land, depending on the current vehicle mode (Multicopter vs Fixed Wing)
-
+* Fixed Wing / VTOL significant TECS improvements
```
Comment (hamishwillee) [inline en/releases/1.12.md:100]: Which are, and have what docs impacts?

File: en/releases/1.12.md [hunk 6]
```diff
@@ -78,3 +107,57 @@
...
+  * [**BACKPORT**] stm32:Ethernet Fix too big frames
+* Boot up stability for V5-V6X ensuring the LSE (RTC) oscillator is started
+  * [**BACKPORT**] stm32h7:lse fix Kconfig help text
+  * [**BACKPORT**] stm32f7:lse Use Kconfig values directly
+  * [**BACKPORT**] stm32h7:Add DBGMCU
+  * [**BACKPORT**] stm32f7:Add option to auto select LSE CAPABILITY
+  * [**BACKPORT**] stm32h7:Add option to auto select LSE CAPABILITY  
+
+    :::note
+      This Knob will cycle through the correct*
```
Comment (hamishwillee) [inline en/releases/1.12.md:126]: What knob? This looks irrelevant and the line above duplicates
